webui/react/src/hooks/useExperimentTags.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
+
+import { patchExperiment } from 'services/api';
+
+export interface UseExperimentTagsInterface {
+  handleTagListChange: (id: number) => (oldTag: string, newTag: string) => void
+  handleTagListCreate: (id: number) => (tag: string) => void;
+  handleTagListDelete: (id: number) => (tag: string) => void;
+}
+
+const useExperimentTags = (callbackFn?: (() => void)): UseExperimentTagsInterface => {
```

Comment:
**suggestion:** How about passing in the callback as an object with a key of something like `onUpdate` to make it a little bit easier to figure out what needs to be passed into the hook. Something like:
```
const useExperimentTags = ({ onUpdate: () => void }): UseExperimentTagsInterface => {
```